src/System.Runtime.Extensions/tests/System/Runtime/Versioning/VersioningHelperTests.cs

```diff
@@ -10,7 +10,8 @@ namespace System.Runtime.Versioning.Tests
     public static class VersioningHelperTests
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "r3 suffix is hardcoded in .Net Core and return code is not documented")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot, 
+            "r3 suffix is hardcoded in .Net Core and return code is not documented")]
```

Comment:
@stephentoub @danmosemsft Changed the test to check NotNull for UWP. Good to merge ?